Tools/clinic/clinic.py

```diff
@@ -5520,7 +5520,7 @@ def format_docstring(self) -> str:
         # Guido said Clinic should enforce this:
         # http://mail.python.org/pipermail/python-dev/2013-June/127110.html
 
-        lines = f.docstring.splitlines()
+        lines = [line for line in f.docstring.split('\n')]
```

Comment:
`splitlines()` was an incorrect suggestion (sorry!) but the comprehension should still be unnecessary here, since `.split()` already creates a list:

```suggestion
        lines = f.docstring.split('\n')
```

Reply:
Yes, that should work!